src/main/java/Duke.java

```diff
@@ -1,10 +1,136 @@
+import java.util.Scanner;
+import java.util.Arrays;
+
 public class Duke {
-    public static void main(String[] args) {
+    private static final Task[] userTasks = new Task[100];
+    private static int taskCount = 0;
+
+    public static void printGreetMessaage() {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        System.out.println("____________________________________________________________");
+        System.out.println("\tHello! I'm Duke");
+        System.out.println("\tWhat can I do for you?");
+        System.out.println("____________________________________________________________");
+    }
+
+    public static void printByeMessage() {
+        System.out.println("____________________________________________________________");
+        System.out.println("\tBye. Hope to see you again soon!");
+        System.out.println("____________________________________________________________");
+    }
+
+    public static void printAcknowledgeMessage(Task task) {
+        System.out.println("____________________________________________________________");
+        System.out.println("Got it. I've added this task: ");
+        System.out.println("\t[" + task.getType() + "]" + "[✗]" + task.getTaskName());
+        System.out.println("Now you have " + taskCount + " tasks in the list.");
+        System.out.println("____________________________________________________________");
+    }
+
+    public static void printTaskList(int taskCount, Task[] userTasks) {
+        System.out.println("____________________________________________________________");
+        System.out.println("Here are the tasks in your list:");
+        if (taskCount == 0) {
+            System.out.println("\tThe task is empty. Please add a task");
```

Comment:
Great use of error handling and making the program as user friendly as possible. Perhaps in future iterations this could be made using exception handling to improve code structure.